### PR TITLE
Improvement/page structure

### PR DESCRIPTION
**Pull request by santostiago**

### CODE REVIEW

File: app/controllers/management/sites_controller.rb [hunk 1]
```diff
@@ -38,8 +38,21 @@ def set_site
...
-    tree = Page.where(site_id: @site.id).select(:id, :name, :parent_id, :position).hash_tree
+    tree = Page.where(site_id: @site.id).select(:id, :name, :parent_id, :position, :enabled).hash_tree
+    formatted_tree = format_tree tree.keys.first, tree.values.first
+    gon.structure = formatted_tree
+  end
+
 
-    gon.structure = tree
+  def format_tree(node_k, node_v)
+    h = {id: node_k.id, name: node_k.name, parent_id: node_k.parent_id, position: node_k.position, enabled: node_k.enabled}
```
Comment (tiagojsag) [inline app/controllers/management/sites_controller.rb]: please use descriptive variable names, not h k v

File: app/views/management/sites/structure.html.erb [hunk 2]
```diff
@@ -6,3 +6,5 @@
 
   <%= link_to 'Save', management_site_site_pages_path, class: 'btn btn-primary' %>
 </div>
+
+<%= console %>
```
Comment (tiagojsag) [inline app/views/management/sites/structure.html.erb]: add it only if env == dev

File: app/controllers/management/sites_controller.rb [hunk 3]
```diff
@@ -38,8 +38,21 @@ def set_site
   end
 
   def build_pages_tree
-    tree = Page.where(site_id: @site.id).select(:id, :name, :parent_id, :position).hash_tree
+    tree = Page.where(site_id: @site.id).select(:id, :name, :parent_id, :position, :enabled).hash_tree
```
Comment (tiagojsag) [inline app/controllers/management/sites_controller.rb]: we probably also need type here (for example, maps cannot be deleted, so the delete button needs to be dependent on the type)

Comment (santostiago) [inline app/controllers/management/sites_controller.rb]: Ok!

File: app/controllers/management/sites_controller.rb (hunk 1 above)
Comment (santostiago) [inline app/controllers/management/sites_controller.rb]: Ok!

File: app/views/management/sites/structure.html.erb (hunk 2 above)
Comment (santostiago) [inline app/views/management/sites/structure.html.erb]: OK!

**Pull request merged**